catalog/mvcc/cataloger_test.go

```diff
@@ -99,7 +99,7 @@ func testVerifyEntries(t testing.TB, ctx context.Context, c catalog.Cataloger, r
 	for _, entry := range entries {
 		ent, err := c.GetEntry(ctx, repository, reference, entry.Path, catalog.GetEntryParams{})
 		if entry.Deleted {
-			if !errors.As(err, &db.ErrNotFound) {
+			if !errors.Is(err, db.ErrNotFound) {
```

Comment:
Ouch, this one is particularly nasty.

Thanks!

Reply:
yes, this is the one that failed the weekend bug fix when you run all the tests